Argo/Functions/catDecoded.swift

```diff
@@ -1,9 +1,32 @@
+/**
+  Remove `Failure` values from an array and unwrap `Success`es
```

Comment:
Is the standard to not have a period at the end of this first line of documentation?